Cleaning up codes, TODOs, and language.

kisaacs opened the pull request:
- Remove finished TODO markers per discussion
- Fix formatting issues in MM-2
- Change "People" to "Participants" in codes
- Reword some codes into (hopefully shorter) sentences.

kisaacs committed:
Removing TODO markers as they are either resolved or about to be.

kisaacs committed:
Trying to fix idention for MM-2.

kisaacs committed:
Absorb significant mental model changes into MM-1.

kisaacs committed:
Outlier code in progress and code sentence editing.

kisaacs committed:
Remove outlier code. Change people to participants.

kisaacs commented:
I wasn't sure what to do with the aggregative marks codes. I'm not sure I understand that what we want to say there in sentence form.

kawilliams commented on codes.md:
```diff
@@ -311,7 +304,7 @@ These codes describe observations about mental models and their formation.
 
-### MM6. Physical objects can also be used to represent data
+### MM6. Physical objects can be used to represent data.
   - This code cross-references mental models, depictions, AND what-is-data,
```
Maybe "this code cross-references the themes mental models, depictions, AND what-is-data..."

kawilliams commented on codes.md:
```diff
@@ -454,12 +442,11 @@ These codes suggest how ordering arises in the mental model
 In these codes, participants infer a bigger picture or task.
 
-### PS1.Participants often want to add additional information, data that isn’t there
+### PS1. Participants often want to add additional data or information.
```
Tense: "wanted"

kawilliams commented on codes.md:
```diff
@@ -550,7 +535,7 @@ further embellishment.
       with "txt" but participant says they stop labeling because of laziness
 
-### D2. People run into constraints with their existing sketch.
+### D2. Participants ran into constraints with their existing sketch.
```
I may want to add Session 030 and 033 -- both had difficulty picking icons/file extensions for the code files that also "went with" the existing icons/encoding. Possibly also people in the junk drawer who drew pens and pencils and then were concerned about differentiating sharpies. 

This is a Katy TODO tomorrow (add 2-3 instances of each of those)

kawilliams committed:
Examples of difficulty encoding the "corner case"

Elaborated on D2 because by "constraints," in addition to "running out of space", I also meant "ran into problems with their encoding choice." These comments from participants only appeared in the file system data set.

kawilliams commented:
I added my tweaks and support to D2 (run into constraints). Merging, and leaving the note about possibly combining D4 (add details) and D6 (elide details)
